### PR TITLE
Once through the whole Spec, fixing many small things

### PR DESCRIPTION
**Pull request by nicholascar**
This PR is trying to fix a whole bunch of structural and presentation issues in the Spec, without altering anything normative/important.

Many of the fixes here are just patching up my bad initial ASCIIDOC conversion...

I have added in some obviously missing parts, e.g. Abstract Tests for the new KML etc. literals added.

This PR doesn't fix many/any substantive issues but really does address a huge number of tiny document formulation issues. Please err on the side of passing this and submitting more PRs to patch the docs further as this PR really transforms the doc completely and will make subsequent PRs easier. 

@situx please could you review the Abstract Tests that link with the literals you added?

@dr-shorthair I changed `inCRS` to `inSRS` so that it's using the more generic form but I've not altered the legacy property `getSRID`

@mperry455 please could you review the structure of the document as per the ToC and see if I've made any structural mistakes such as headings at the wrong level

The rendered output PDF and HTML files are:

* [HTML](https://raw.githack.com/opengeospatial/ogc-geosparql/oncethrough/1.1/spec/oncethrough.html)
* [PDF](https://raw.githack.com/opengeospatial/ogc-geosparql/oncethrough/1.1/spec/oncethrough.pdf) 

I've focused on HTML generation for now so there are errors in the PDF which will need to be patched so please focus on the HTML version for reviews.

These are just for review here (saving you building the files yourself) so I'll delete them once this PR is merged as the auto-build process will recreate them.

I've introduced the profile here:

* <https://raw.githack.com/opengeospatial/ogc-geosparql/oncethrough/1.1/spec/oncethrough.html#_geosparql_standard_structure>

### CODE REVIEW

**Comment by situx** [comment]
I think it would be nice if we were to make all URIs in the specification clickable. Many are dereferenceable anyway.

**Comment by situx** [comment]
I added many links. Hope I got them all

**Comment by situx** [comment]
Maybe it should be a different pull request, but we would need to update the Future Work section, as many of the points raised there are tackled in GeoSPARQL 1.1

**Comment by nicholascar** [comment]
I'll do a very basic tidy-up of the Future Work section - removing things we've actually done - but then yes, we will need other PRs to further update it so that everyone can have their say on what the future really should look like, including steps towards GeoSPARQL 2.0 etc.

**Review by situx — APPROVED** [review]
I added many missing links and corrected some minor issues. Now I think it is in a state which I can approve.

File: 1.1/spec/01-Preamble.adoc [hunk 1]
```diff
@@ -1,65 +1,80 @@
...
+.. Mainz University Of Applied Sciences
+.. Oracle America
+.. Ordnance Survey
+.. Raytheon Company
+.. SURROUND Australia Pty Ltd.
+.. Traverse Technologies, Inc.
+.. US Geological Survey (USGS)
+
+
+=== Submission contact points
```
Comment (mperry455) [inline 1.1/spec/01-Preamble.adoc:35]: Should use iii. section number here

File: 1.1/spec/10-Part-07.adoc [hunk 2]
```diff
@@ -1,94 +1,100 @@
...
 
-==== Parameters
+=== Parameters
 
 The following parameter is defined for the _Topology Vocabulary Extension_ requirements class.
 
 *relation_family*: Specifies the set of topological spatial relations to support.
 
-==== Requirements for Simple Features Relation Family (relation_family=Simple Features)
```
Comment (mperry455) [inline 1.1/spec/10-Part-07.adoc:13]: I like the style of dropping "Requirements for" from the section titles. If this is what we want to do then we also need to drop "Requirements for" from titles in Sections 9, 10 and 11.

**Review by mperry455 — APPROVED** [review]
Looks great. Just a couple of minor comments that I added in-line.

File: 1.1/spec/10-Part-07.adoc (hunk 2 above)
Comment (nicholascar) [reply 1.1/spec/10-Part-07.adoc:13]: Done in Commit https://github.com/opengeospatial/ogc-geosparql/pull/102/commits/72c2a7f59b7e4f1085f06553c4dfeafd0123976a

File: 1.1/spec/01-Preamble.adoc (hunk 1 above)
Comment (nicholascar) [reply 1.1/spec/01-Preamble.adoc]: Fixed in https://github.com/opengeospatial/ogc-geosparql/pull/102/commits/3b128f7cc36e7e2db354395b399e7daca2cc7eb0

**Pull request merged**